com.github.retext_project.retext.yaml

```diff
@@ -0,0 +1,84 @@
+app-id: com.github.retext_project.retext
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+base: com.riverbankcomputing.PyQt.BaseApp
+base-version: 5.15-21.08
+command: retext
+rename-desktop-file: me.mitya57.ReText.desktop
+rename-icon: retext
+rename-appdata-file: me.mitya57.ReText.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=wayland
+  - --filesystem=home
+
+modules:
+  - name: python3-Markdown
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/a4/63/eaec2bd025ab48c754b55e8819af0f6a69e2b1e187611dd40cbbe101ee7f/Markdown-3.2.2-py3-none-any.whl
+        sha256: c467cd6233885534bf0fe96e62e3cf46cfc1605112356c4f9981512b8174de59
+        x-checker-data:
+          type: pypi
+          name: Markdown
+    build-commands:
+      - pip3 install Markdown-3.2.2-py3-none-any.whl
+
+  - name: python3-python-markdown-math
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/4d/36/d16118345ff5f503d4d1c708e8427ed2213f2af7aaffac9d59010e665b5c/python_markdown_math-0.8-py3-none-any.whl
+        sha256: c685249d84b5b697e9114d7beb352bd8ca2e07fd268fd4057ffca888c14641e5
+        x-checker-data:
+          type: pypi
+          name: Markups
+    build-commands:
+      - pip3 install python_markdown_math-0.8-py3-none-any.whl --prefix=${FLATPAK_DEST}
+
+  - name: python3-Markups
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/94/5f/889877bf7f06eb581f88754d06d0472af566a1f196df46ea6f60569d2f70/Markups-3.1.2-py3-none-any.whl
+        sha256: 40058f2ea249e24f1eba5542718f2e9c0a427923af488927094938c16607eb42
+        x-checker-data:
+          type: pypi
+          name: Markups
+    build-commands:
+      - pip3 install Markups-3.1.2-py3-none-any.whl --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST}
+
+  - name: python3-pyenchant
+    buildsystem: simple
+    sources:
+      - type: file
+        url: https://files.pythonhosted.org/packages/14/4b/6f7a3f2bb1e2fa4d3007126578cae0b9910ff46c4957bef5bd4b92733011/pyenchant-3.0.1.tar.gz
+        sha256: 1bd26a644abf80196a9de3f2d820ebafb7e7f78385e392ce77cb1552f164d559
+    build-commands:
+      - pip3 install --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} pyenchant
+
+  - name: python3-docutils
```

Comment:
I highly recommend just running `[flatpak-pip-generator](https://github.com/flatpak/flatpak-builder-tools/blob/master/pip/flatpak-pip-generator)` and having it output a separate file. That allows updating modules to be a trivial single command. As-is you have to manually edit the main file.

Reply:
That is indeed a lot more convenient. Thanks!